capstone/src/main/webapp/youtubeScript.js

```diff
@@ -162,15 +202,83 @@ function showAvailableAttributes() {
     label.appendChild(document.createTextNode(attribute));
     avaiableAttributesElement.appendChild(checkbox);
     avaiableAttributesElement.appendChild(label);
-    avaiableAttributesElement.appendChild (document.createTextNode (" "));
+    avaiableAttributesElement.appendChild(document.createTextNode (" "));
```

Comment:
nit: Let's delete the extra space after `createTextNode` and on line 264 & 282

Reply:
addressed in "remove extra space